File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.c
int mostWordsFound(char ** sentences, int sentencesSize){
    int i,j;
    int max = 0,tmp;
    
    for(i = 0; i < sentencesSize ; ++i)
    {
        j = 0;
        tmp = 1;
        while(sentences[i][j])
        {
            if(sentences[i][j] == ' ')
                tmp++;
            j++;
        }
        if(tmp > max)
            max = tmp;
    }
    return max;
}

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int i,j,max = 0;
        for(i = 0; i < sentences.length; i++)
        {
            String[] str = sentences[i].split(" ");
            for(j = 0; j < str.length; j++);
            if(j > max)
                max = j;
        }
        return max;
    }
}